lib/index.js

```diff
@@ -32,6 +32,17 @@ ytdl.filterFormats = util.filterFormats;
 ytdl.cache = cache;
 
 
+/**
+ * checks wether the input string includes a valid id
+ *
+ * @param {String} string
+ */
+ytdl.validateLink = function(string) {
```

Comment:
Would rather place this inside of `util.js` and export it here like the other util functions.